src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+
+        truck.doWork();
+        bulldozer.doWork();
+        excavator.doWork();
+        truck.stopWork();
+        bulldozer.stopWork();
+        excavator.stopWork();
```

Comment:
According to the task, you should create an array with your machines and call their methods in the loop.
<img width="806" alt="image" src="https://github.com/mate-academy/jv-oop/assets/32339836/cc9b176a-00f1-43c7-a816-e134b606ec41">